docs/how-to/h-config-backup-restore.md

```diff
@@ -0,0 +1,117 @@
+# Configuration backup and restore
+
+Charmed Kafka's configuration is distributed using [Charmed ZooKeeper](https://charmhub.io/zookeeper?channel=3/stable).
+A Charmed ZooKeeper backup can be stored on any S3-compatible storage.
+S3 access and configurations are managed with the [`s3-integrator` charm](https://charmhub.io/s3-integrator).
+
+This guide will teach you how to deploy and configure the `s3-integrator` charm for [AWS S3](https://aws.amazon.com/s3/), send the configurations to the Charmed ZooKeeper application, and finally manage your Charmed ZooKeeper backups.
+
+## Summary
+
+- [Configure `s3-integrator`](#heading--s3)
+- [Create a backup](#heading--create)
+- [List backups](#heading--list)
+- [Restore a backup](#heading--restore)
+
+<a href="#heading--s3"><h2 id="heading--s3">Configure `s3-integrator`</h2></a>
```

Comment:
I believe the HTML syntax is needed for the `id` attribute, otherwise the anchor links from the summary would not work. Do you know an alternative syntax to keep this behavior?

Reply:
I'm not entirely sure what you mean. Other pages I've seen have the default headings and they work as anchors (for example - https://discourse.charmhub.io/t/charmed-kafka-k8s-documentation-how-to-deploy/13266#juju-controller-setup-2). Although I didn't test it by linking from inside the content itself. 
But:
1. This page is not that big, I think we can do without links in the list.
2. I do remember hearing about a bug fix for Discourse anchors recently (like last week). I wonder whether it was about the issue we are trying to solve here.

Reply:
Ok, we can drop the summary thing then, and that also takes care of the thread about moving the paragraph

Reply:
Fixed in a45eb7550f57